src/components/tree/infoPanels/helpers.js

```diff
@@ -0,0 +1,23 @@
+import { getTraitFromNode } from "../../../util/treeMiscHelpers";
+  import { numericToCalendar } from "../../../util/dateHelpers";
```

Comment:
Remove extra indent:

```suggestion
import { getTraitFromNode } from "../../../util/treeMiscHelpers";
import { numericToCalendar } from "../../../util/dateHelpers";
```

Reply:
Fixed in [force push](https://github.com/nextstrain/auspice/compare/962a2e24e46a5a6f9b28a9157deebd662d9d08db..399c50181ab1ed35a88c1a6abc95b91193502eb0)